backend/tests/test_auth_endpoints.py

```
@@ -0,0 +1,77 @@
+    Tenant = None
+
+class TestAuthEndpoints(unittest.TestCase):
+    def setUp(self):
+        if not app or not db:
+            self.skipTest("Flask app or DB not available for testing.")
+            return
+
+        app.config['TESTING'] = True
+        app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///:memory:'
+        # For JWT, if needed for these tests specifically
+        app.config['JWT_SECRET_KEY'] = 'test-secret-key'
```

Ruff (0.11.9)

Possible hardcoded password assigned to: "JWT_SECRET_KEY"

(S105)